scripts/update_covid19.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+import io
+import json
+import os
+import re
+
+import boto3
```

Comment:
should `boto3` be in `requirements.txt` too?

Reply:
of course you're right! I had removed it for the initial testing on AWS to lighten up the zip file, as boto3 is already installed up there. I'll fix